### PR TITLE
[New3D] support velodyne_msgs/VelodyneScan

### LINKED ISSUE #4540: Add velodyne_msgs/VelodyneScan support to new 3D panel

### PR DESCRIPTION
**Pull request by jtbandes**
**User-Facing Changes**
None (feature was present in old 3d panel)

**Description**
Adds velodyne_msgs/VelodyneScan support which was missing from new3d. Closes #4540.

### CODE REVIEW

File: packages/studio-base/src/panels/ThreeDeeRender/renderables/VelodyneScans.ts [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+      const pickingMaterial = createPickingMaterial(settings);
+      const instancePickingMaterial = createInstancePickingMaterial(settings);
+      const points = createPoints(
+        topic,
+        pointCloud.pose,
+        geometry,
+        material,
+        pickingMaterial,
+        instancePickingMaterial,
+      );
```
Comment (jtbandes) [inline packages/studio-base/src/panels/ThreeDeeRender/renderables/VelodyneScans.ts:237]: @jhurliman is there a better way you'd recommend than to duplicate this logic from the PointCloudsAndLaserScans?

**Comment by snosenzo** [comment]
Can you summarize some of the changes you made to the `PointCloudsAndLaserScans` file?

**Comment by jtbandes** [comment]
Mostly just moved code from the scene extension into the renderable.

**Comment by snosenzo** [comment]
> Mostly just moved code from the scene extension into the renderable.

What's the benefit of that? just want to understand if I should also be looking out and doing that for stuff like `foxglove.Grid`

**Review by snosenzo — APPROVED** [review]
Oh I see, you put stuff in there to be able to use it w/ the velodyne scans, sorry for the confusion. Just thought it was a lot of noise to refactor initially
